app/Mails/Smtp/BladeMailTemplateResolver.php

```diff
@@ -20,14 +20,16 @@ public function __construct(
         private ViewFactory $view,
     ) {
         $this->templates = [
-            MailTemplate::INTEGRATION_ACTIVATED->value => 'integration.activated',
+            MailTemplate::INTEGRATION_CREATED->value => 'integration.uitpas.created',
             MailTemplate::ORGANISATION_UITPAS_REQUESTED->value => 'UdbOrganizer.requested',
+            MailTemplate::INTEGRATION_ACTIVATED->value => 'integration.uitpas.activated',
```

Comment:
Why is this linked with `integration.uitpas.activated` and not `integration.activated`? It feels like you are reusing the UiTPAS specific mails now also for the normal integrations?

Reply:
I checked all templates, the activated template is indeed shared. It is currently not sent yet, but will be used once we remove the Mailjet integration, which I am working on this week.